test/resources/systemd.resource

```diff
@@ -0,0 +1,81 @@
+*** Settings ***
+Documentation    Keywords for interacting with systemd
+
+Library    Process
+Library    String
+Resource   ../resources/microshift-host.resource
+
+
+*** Keywords ***
+Get Systemd Setting
+    [Documentation]  Fetch one setting from systemd for the named unit.
+    ...  Take care to get the unit_name value _exactly_ right, or
+    ...  systemd will report a default value without reporting any error
+    ...  or warning.
+    [Arguments]  ${unit_name}  ${property}
+
+    ${stdout}    ${rc}=    Execute Command
+    ...    systemctl show --property=${property} --value ${unit_name}
+    ...    sudo=True    return_rc=True
+    Should Be Equal As Integers    ${rc}    0
+    Should Not Be Empty    ${stdout}
+
+    ${result}=  Strip String  ${stdout}
+    RETURN  ${result}
+
+Systemctl    # robocop: disable=too-long-keyword
+    [Documentation]  Run a systemctl command on the microshift host.
+    ...  The intent is to start, stop, or restart a service. Other
+    ...  commands should be implemented separately.  When the verb is
+    ...  "start" or "restart", this keyword will wait for the unit
+    ...  to report that it is "running". When the verb is "stop", this
+    ...  keyword will wait for the unit to report that it is "dead".
+    [Arguments]  ${verb}  ${unit_name}
+
+    IF  "${verb}" in {"restart", "start"}
+        ${state}=  Set Variable  running
+    ELSE
+        ${state}=  Set Variable  dead
+    END
+
+    ${stdout}  ${stderr}  ${rc}=  Execute Command
+    ...    systemctl ${verb} ${unit_name}
+    ...    sudo=True
+    ...    return_stdout=True
+    ...    return_stderr=True
+    ...    return_rc=True
+    Log  ${stdout}
+    Log  ${stderr}
+    IF  ${rc} != 0
+        ${status_text}=  Execute Command
+        ...    systemctl status ${unit_name}
+        ...    sudo=True
+        ...    return_stdout=True
+        Log  ${status_text}
+        ${log_text}=  Execute Command
```

Comment:
This is already covered by sos report, isnt it? It doesnt even matter if the test succeeds or not (it fails here, though). You are going to get a huge output from this, mixed with the other tests/keywords output.

Reply:
We may want to cut this down, but I found it useful in debugging to see the logs from the process at the time of the command inline. I could make it 20 lines?